comm/src/main/scala/coop/rchain/comm/transport/TcpTransportLayer.scala

```diff
@@ -46,7 +46,7 @@ class TcpTransportLayer(port: Int, cert: String, key: String, maxMessageSize: In
   private def certInputStream = new ByteArrayInputStream(cert.getBytes())
   private def keyInputStream  = new ByteArrayInputStream(key.getBytes())
 
-  private val streamObservable = new StreamObservable(100, tempFolder)
+  private val streamObservable = new StreamObservable(1000, tempFolder)
```

Comment:
those values are arbitrary, please create ticket in jira to fine tune this at some point (maybe configuration flag would be a good opiton here)

Reply:
I will prepare this value as a constructor parameter and create a ticket.